src/frontend/pages/question-page/question-page.js

```diff
@@ -1,11 +1,9 @@
-import { useState } from "react";
 import { Footer, Navbar} from "../../components/index";
 import { useQuestion } from "../../contexts/index";
 import "./question-page.css";
 
 export function QuestionPage(){
-
-    const { nextQuestionHandler, category, categoryQuestions, seconds, 
+    const { nextQuestionHandler, category, categoryQuestions, seconds, error,
         QuitQuiz, curqueIndex, selectOptionHandler, viewResult, selectedOption} = useQuestion();
```

Comment:
Use camelCase for 'currQuesIndex' so that it's easily understandable that it is current question's index.